core/server/common/src/main/java/alluxio/master/journal/FileSystemMergeJournalContext.java

```diff
@@ -77,8 +74,12 @@ public synchronized void append(JournalEntry entry) {
     mJournalEntryMerger.add(entry);
     List<JournalEntry> journalEntries = mJournalEntryMerger.getMergedJournalEntries();
     if (journalEntries.size() >= MAX_LOGGING_ENTRIES) {
-      SAMPLING_LOG.warn("MergeJournalContext has " + journalEntries.size()
-          + " entries, over the limit of " + MAX_LOGGING_ENTRIES);
+      LOG.warn("MergeJournalContext has " + journalEntries.size()
```

Comment:
i assume this should be rare enough that even not using sampling LOG would be fine?

Reply:
yes because:
1. this is not supposed to happen 
2. even it happens, all journals will be flushed and the # of journal entry will become 0 again